docs/guides/high-availability/single-cluster/building-blocks.adoc

```diff
@@ -0,0 +1,53 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Building blocks single-cluster deployments"
+summary="Learn about building blocks and suggested setups for single-cluster deployments."
+tileVisible="false" >
+
+The following building blocks are needed to set up a single-cluster deployment.
+
+The building blocks link to a blueprint with an example configuration.
+They are listed in the order in which they need to be installed.
+
+include::../partials/blueprint-disclaimer.adoc[]
+
+[#single-cluster-blocks-prerequisites]
+== Prerequisites
+
+* Understanding the concepts laid out in the <@links.ha id="single-cluster-concepts"/> {section}.
+
+
+[#single-cluster-blocks-low-latency]
+== Multiple availability-zones with low-latency connection
+
+Ensures that synchronous replication is available for both the database and {project_name} clustering.
+
+*Suggested setup:* A {kubernetes} cluster consisting of two or more AWS Availability Zones within the same AWS Region.
+
+*Not considered:* {kubernetes} clusters spread across multiple regions on the same or different continents, as it would increase the latency and the likelihood of network failures.
+Synchronous replication of databases as services with Aurora Regional Deployments on AWS is only available within the same region.
+
+
+[#single-cluster-blocks-kubernetes]
+== {kubernetes} cluster
+
+*Suggested setup:* Red Hat OpenShift Service on AWS (ROSA) deployed across multiple availability-zones with at least one worker node in each zone.
```

Comment:
The topic above says `A {kubernetes} cluster consisting of two or more AWS Availability Zones within the same AWS Region.` 
This section seems duplicated.